action/README.md

```diff
@@ -74,6 +74,86 @@ jobs:
               ${{ steps.update.outputs.flux_version }}
 ```
 
+### Push Kubernetes manifests to container registries
+
+Example workflow for publishing Kubernetes manifests bundled as OCI artifacts to GitHub Container Registry:
+
+```yaml
+name: push-artifact-staging
+
+on:
+  push:
+    branches:
+      - 'main'
+
+permissions:
+  packages: write # needed for ghcr.io access
+
+env:
+  OCI_REPO: "oci://ghcr.io/my-org/manifests/${{ github.event.repository.name }}"
```

Comment:
There is a surprising amount of nuance in the configuration provided here without much commentary. That's probably the right character for this type of doc, but we may want to add more details.

We probably ought to include a link to some docs that elaborate on how to create a free-standing package repo, or users will trip and may give up when they try using it with an error like this:

<img width="1029" alt="Screen Shot 2022-08-09 at 1 26 00 PM" src="https://user-images.githubusercontent.com/3286998/183718677-f9f626ec-951d-4345-85a0-a8ac23be211c.png">